### PR TITLE
add ANOVA description and UI

### PR DESCRIPTION
**Pull request by dropmann**
- Also changed some UI description formatting

### CODE REVIEW

File: inst/silky/anova.a.yaml [hunk 1]
```diff
@@ -15,8 +15,154 @@ options:
...
+
+    - name: corrHolm
+      title: Holm
+      type: Bool
+      default: false
+      description: >
+        Post Hoc Tests Holm Correction
+
+    - name: margMeans
+      title: Marinal Means
```
Comment (jonathon-love) [inline inst/silky/anova.a.yaml:89]: spelling

File: inst/silky/anova.a.yaml [hunk 2]
```diff
@@ -15,8 +15,154 @@ options:
...
+        Display Descriptive statistics
+
+    - name: estEffSize
+      title: Estimates of effect size
+      type: Bool
+      default: false
+      description: >
+        Display Estimates of effect size
+
+    - name: homoTests
```
Comment (jonathon-love) [inline inst/silky/anova.a.yaml:122]: not PC!

Comment (dropmann) [inline inst/silky/anova.a.yaml:122]: Any suggestions as to a more appropriate name?

Comment (jonathon-love) [inline inst/silky/anova.a.yaml:122]: she'll be right, we'll come back to this later.

jonathon

**Pull request merged**